packages/opencensus-exporter-jaeger/test/test-jaeger.ts

```diff
@@ -80,6 +80,18 @@ describe('Jaeger Exporter', () => {
     it('should encode as thrift', () => {
       return tracer.startRootSpan({name: 'root-s01'}, (rootSpan) => {
         const span = tracer.startChildSpan('child-s01');
+        span.addAttribute('testBool', true);
+        span.addAttribute('testString', 'here');
+        span.addAttribute('testNum', 3.142);
+        span.addAnnotation(
+            'something happened',
+            Date.now(),
+            {
+              'error': true,
+              'severity': 'high',
+              'priority': 4,
+            },
+        );
```

Comment:
Is it possible to tell that these attributes and annotations were written as Thrift tags?

Reply:
I'll try and improve the test.